src/main/java/com/lunark/lunark/security/AuthRequest.java

```diff
@@ -1,7 +1,16 @@
 package com.lunark.lunark.security;
 
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
 public class AuthRequest {
+    @NotNull
```

Comment:
I think @NotEmpty checks for null as well so this might not be needed.

Reply:
Good catch. I have just checked and it is indeed the case.